documentation/docs/05-load.md

```diff
@@ -264,6 +219,56 @@ Setting the same header multiple times (even in separate `load` functions) is an
 
 You cannot add a `set-cookie` header with `setHeaders` — use the [`cookies`](/docs/types#sveltejs-kit-cookies) API in a server-only `load` function instead.
 
+### Inputs exclusive to shared load functions
+
+In addition to the common inputs, a `load` function that runs on both the server and the client has the following inputs:
+
+#### data
+
+Very rarely, you might need both a `+page.js` and a `+page.server.js` (or the `+layout` equivalent). In these cases, the `data` for `+page.svelte` comes from `+page.js`, which in turn receives `data` from the server:
+
+```js
+/// file: src/routes/my-route/+page.server.js
+/** @type {import('./$types').PageServerLoad} */
+export function load() {
+	return {
+		a: 1
+	};
+}
+```
+
+```js
+/// file: src/routes/my-route/+page.js
+// @filename: $types.d.ts
+export type PageLoad = import('@sveltejs/kit').Load<{}, { a: number }>;
+
+// @filename: index.js
+// ---cut---
+/** @type {import('./$types').PageLoad} */
+export function load({ data }) {
+	return {
+		b: data.a * 2
+	};
+}
+```
+
+```svelte
+/// file: src/routes/my-route/+page.svelte
+<script>
+	/** @type {import('./$types').PageData} */
+	export let data;
+
+	console.log(data.a); // `undefined`, it wasn't passed through in +page.js
+	console.log(data.b); // `2`
+</script>
+```
+
+In other words `+page.server.js` passes `data` along to `+page.js`, which passes `data` along to `+page.svelte`.
+
+### Inputs exclusive to server-only load functions
+
+In addition to the common inputs, the server-only `load` function provides a few more arguments which it inherits from `RequestEvent`. Namely these are `getClientAddress`, `cookies`, `locals`, `platform` and `request`. See the `RequestEvent` docs for more info.
```

Comment:
Can we add a link to `RequestEvent`'s docs from here since it's called out?

Reply:
Links are applied automatically

<img width="365" alt="image" src="https://user-images.githubusercontent.com/1162160/193077010-fd5b8780-8f04-4d7a-a27c-c69fc65e38f0.png">